code/modules/mob/living/say.dm

```diff
@@ -93,6 +93,11 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(!message || message == "")
 		return
 
+	var/static/regex/netspeak = regex("\\b(lol|brb|wtf|idk|omg|lmao|btw|ikr|rofl|tbh|nvm|thx|afaik|ooc|ic|pls|plz)\\b", "i")

```

Comment:
does this actually ensure the regex is only compiled once?

Also I suspect a chain of findtext calls will actually be faster.

third this should be configurable list in config

@MrStonedOne are you happy for a wordfilter to be applied?

Reply:
Make it a configurable list and kick the bucket down to the headmins, but this would need to be changed as static vars are inited with global vars before anything else happens, including config loading.

Reply:
I don't know how to check if the regex is only compiled once.

But I'll profile this vs findtext and rework it into a config when I have time, if you are willing to merge this once those improvements have been made. Otherwise I won't waste my time.

Reply:
Im not opposed to it

make sure you add a config option to globally enable/disable it as well, or just use the empty state of the list to skip any checks

Reply:
Turned this into a config.